### PR TITLE
Add default scope to batch url

### PR DESCRIPTION
**Pull request by jsaun**
Part of https://github.com/microsoft/CromwellOnAzure/issues/532 
Missed in https://github.com/microsoft/ga4gh-tes/pull/608

### CODE REVIEW

File: src/TesApi.Web/AzureProxy.cs [hunk 1]
```diff
@@ -101,7 +101,7 @@ public AzureProxy(IOptions<BatchAccountOptions> batchAccountOptions, BatchAccoun
             else
             {
                 location = batchAccountInformation.Region;
-                batchClient = BatchClient.Open(new BatchTokenCredentials(batchAccountInformation.BaseUrl, () => GetAzureAccessTokenAsync(CancellationToken.None, "https://batch.core.windows.net/")));
+                batchClient = BatchClient.Open(new BatchTokenCredentials(batchAccountInformation.BaseUrl, () => GetAzureAccessTokenAsync(CancellationToken.None, "https://batch.core.windows.net//.default")));
```
Comment (BMurri) [inline src/TesApi.Web/AzureProxy.cs:104]: This one should only be one slash.

**Review by BMurri — APPROVED** [review]
Thanks for catching this

File: src/TesApi.Web/AzureProxy.cs (hunk 1 above)
Comment (jsaun) [reply src/TesApi.Web/AzureProxy.cs:104]: Matt pointed out that double slash is the convention for public cloud actually https://github.com/microsoft/CromwellOnAzure/pull/748#discussion_r1495027845

Comment (BMurri) [reply src/TesApi.Web/AzureProxy.cs:104]: That only applies to the public cloud resource manager (control plane). It does not apply to any data-plane system, nor to the resource manager on the sovereign clouds.

**Pull request merged**